static/beta/prod/navigation/quay-navigation.json

```diff
@@ -3,6 +3,26 @@
   "title": "Quay.io",
   "description": "Quay builds, analyzes, distributes your container images.",
   "navItems": [
+    {
+      "id": "quayOverview",
+      "appId": "quay-ui-plugin",
+      "title": "Overview",
+      "icon": "QuayIoIcon",
+      "description": "Overview information about quay.io",
+      "href": "/quay/overview"
+      "permissions": [
+        {
+            "method": "apiRequest",
+            "args": [
+                {
+                    "url": "/beta/apps/quay/plugin-manifest.json",
```

Comment:
Is this still required? Is the Quay plugin still supposed to be hidden from some users? Just curious.

Reply:
Not sure, I just figured it's best to leave it the same as the other navs.

Reply:
@Marcusk19 can I request a follow-up to this? I vaguely remember this was done during the initial release to ensure the nav items show when the deployment is ready. The quay plugins are now in production so this API check will always be true. Which makes the API calls obsolete and wastes resources.